lemminx-maven/pom.xml

```diff
@@ -22,7 +22,7 @@
 		<lemminx.maven.indexDirectory>${project.build.directory}/test-index</lemminx.maven.indexDirectory>
 		<maven.repo.local>${settings.localRepository}</maven.repo.local>
 		<junit-jupiter.version>5.7.0</junit-jupiter.version>
-		<lemminx.version>0.15.0</lemminx.version>
+		<lemminx.version>0.15.1-SNAPSHOT</lemminx.version>

```

Comment:
Probably worthwhile to upgrade to LemMinX 0.16.2 now

Reply:
Updated to 0.16.2